### DISCUSSION TITLE
How to parse a string with break lines ("\n")?

### DISCUSSION THREAD

**Discussion by AnthonyLzq** [neoclide/coc.nvim#3794] [Q&A] [answered]
I followed the example from the docs, but it doesn't seem to work:

![image](https://user-images.githubusercontent.com/43073718/166306602-1b4f330f-be30-4bed-915e-14d10fa73b2a.png)

Which config should I have to parse the '\n' correctly?

**Comment by chemzqm** [comment, marked as answer]
See https://github.com/neoclide/coc.nvim/issues/3720 for markdown used with coc.nvim
